### ISSUE TITLE
Now redundant category for wave vector modulation?

### ISSUE THREAD

**Issue by nautolycus** [COMCIFS/magnetic_dic#76]
There is a category ATOM_SITE_FOURIER_WAVE_VECTOR that is essentially an extension to the msCIF dictionary. These are items that were needed in some magnetic structures but are not specifically related to magnetic properties. This category now exists in the latest msCIF DDLm version on GitHub. Should it now be removed from magCIF (which formally imports cif_ms.dic)?

**Comment by brantonc** [comment]
I don’t have a strong opinion.  As long as practitioners can use the tag, it doesn’t really matter to me which dictionary it rests in.

Best wishes,
Branton

From: nautolycus ***@***.***>
Sent: Wednesday, March 13, 2024 11:31 AM
To: COMCIFS/magnetic_dic ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [COMCIFS/magnetic_dic] Now redundant category for wave vector modulation? (Issue #76)


There is a category ATOM_SITE_FOURIER_WAVE_VECTOR that is essentially an extension to the msCIF dictionary. These are items that were needed in some magnetic structures but are not specifically related to magnetic properties. This category now exists in the latest msCIF DDLm version on GitHub. Should it now be removed from magCIF (which formally imports cif_ms.dic)?

**Comment by jamesrhester** [comment]
I agree that the category definition can be removed from the magnetic dictionary as it is available in the MS dictionary.

**Comment by vaitkus** [comment]
The category was removed by merging PR #85.
